components/org.wso2.carbon.identity.application.authenticator.push/src/main/java/org/wso2/carbon/identity/application/authenticator/push/PushAuthenticator.java

```diff
@@ -125,7 +130,13 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
 
             if (deviceList.size() == 1) {
                 RequestSender requestSender = new RequestSenderImpl();
-                requestSender.sendRequest(request, response, deviceList.get(0).getDeviceId(), sessionDataKey);
+                Optional<String> optionalAdditionalInfo = getAdditionalInfo(request, response, sessionDataKey);
+                if (optionalAdditionalInfo.isPresent()){
+                    additionalInfo = optionalAdditionalInfo.get();
+                    log.debug(String.format("Adding additional info %s to the notification", additionalInfo));
```

Comment:
wrap inside log.isDebugEnabled. What sort of information is sent in the optionalAdditionalInfo? (basically are these sensitive information that should be directly logged?)

Reply:
According to the considered requirement `optionalAdditionalInfo` are not sensitive info. It can have additional info which can further explain the authentication request.
eg: In OB context, we are sending what exact permission are required,validity period for the consent etc.